shared/src/PayloadEncoder.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Text;
+
+namespace Microsoft.Azure.Devices.Shared
+{
+    /// <summary>
+    /// This class specifies the byte encoding for the the payload.
+    /// </summary>
+    /// <remarks>
+    /// The encoder is responsible for encoding all of your objects into the corrent bytes for the <see cref="PayloadConvention"/> that uses it.
+    /// <para>
```

Comment:
I've not found the `<para>` tag to actually be necessary. Intellisense and web docs both seem to treat whitespace as you'd expect. Have you found otherwise?

Reply:
I'll let @jamdavi comment on this, since you've worked on most (if not all :)) of the doc comments.

Reply:
Let me test this assertion. IIRC it might treat whitespace as new lines, but it might not create a new <p> tag in the docs.

Reply:
You can test it with Intellisense immediately. For web docs, all we have to do is find an existing case where we have newlines and see what the docs look like. I'm pretty sure I did that recently and saw it worked fine.

Check out `Device.ParentScopes` as an example: <https://docs.microsoft.com/en-us/dotnet/api/microsoft.azure.devices.device.parentscopes?view=azure-dotnet#Microsoft_Azure_Devices_Device_ParentScopes>